README.md

```diff
@@ -86,7 +76,3 @@ You can cite the `nf-core` publication as follows:
 >
 > _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
 > ReadCube: [Full Access Link](https://rdcu.be/b1GjZ)
```

Comment:
If not too much work you can add references to the components of the pipelines, to make it easier for users to find the corresponding tools it is using.

Reply:
I guess pypgatk is the most important one here

Reply:
@Leon-Bichmann pypgatk hasn't been published.